.github/workflows/test.yml

```diff
@@ -40,21 +40,16 @@ jobs:
           key: ${{ runner.os }}-python-${{ env.pythonLocation }}-${{ hashFiles('pyproject.toml') }}-root
       - name: Install poetry
         if: steps.cache.outputs.cache-hit != 'true'
-        # TODO: remove python -m pip install --force git+https://github.com/python-poetry/poetry-core.git@ad33bc2
-        # once there's a release of Poetry 1.2.x including poetry-core > 1.1.0a6
-        # Ref: https://github.com/python-poetry/poetry-core/pull/188
         run: |
           python -m pip install --upgrade pip
-          python -m pip install --force git+https://github.com/python-poetry/poetry-core.git@ad33bc2
-          python -m pip install "poetry==1.2.0a2"
-          python -m poetry plugin add poetry-version-plugin
+          python -m pip install "poetry==1.3.1"
```

Comment:
I looked into it, and poetry-version-plugin is not compatible with the latest poetry. So you would need to install `1.2.0b1` which is the last working version (and revert back to `plugin add`). If you want to use the latest version of Poetry, poetry-version-plugin should be removed, maybe the workflow will still work since I think poetry added support for `__version__` but I'm not sure. So will have to be tested. The reason the tests pass right now is that it is using cache.

Reply:
I deprecated poetry-version-plugin and change the way of loading __version__
Now version is in poetry